src/main/java/com/dsec/backend/service/UserService.java

```diff
@@ -126,22 +135,98 @@ public UserInfoDTO login(LoginInfoDTO dto, HttpServletResponse response) {
 	}
 
 	@Override
-	public UserInfoDTO getUser(Jwt user) {
+	public ResponseEntity<?> getAllUsers() throws JsonProcessingException {
+		final ObjectMapper mapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+		int i = 0;
+		String responseBody = new String();
+		for (UserModel userModel : userRepository.findAll()) {
+			String tmpString = "";
+			if(i != 0)
+				tmpString = ",";
+			tmpString = tmpString + ("\""+i+"\":"+mapper.writeValueAsString(new UserInfoDTO(userModel.getId(), userModel.getEmail(), userModel.getFirstName(), userModel.getLastName(), userModel.getUserRole())));
+			responseBody = responseBody + tmpString;
+			i++;
+		}
```

Comment:
Please do not use ObjectMapper directly. Spring will implicitly use it to serialise responses. I tried to use it in creation of a jwt and it did not work as expected. Its usage is not common and rarely needed. There are better ways to achive what you wanted to achieve here.